apps/website-25/src/components/courses/exercises/Exercise.stories.tsx

```diff
@@ -1,56 +0,0 @@
-import type { Meta, StoryObj } from '@storybook/react';
```

Comment:
Maybe worth having a story to show course editors how it's used e.g. with a recordId and sample exercise?